src/orchestrator-tracing/src/lib.rs

```diff
@@ -92,6 +92,48 @@ pub struct TracingCliArgs {
     /// Only respected when `--log-format` is `text`.
     #[clap(long, env = "LOG_PREFIX")]
     pub log_prefix: Option<String>,
+    /// The max number of tracing spans to queue before dropping.
+    #[clap(
+        long,
+        env = "OPENTELEMETRY_MAX_BATCH_QUEUE_SIZE",
+        default_value = "2048",
```

Comment:
Are these values the BatchConfig defaults?

Reply:
This one I think is 4x! I def think we should have comments explaining if they are the `BatchConfig` defaults or some larger value! 

Lets starts with 4x queue size, 4x batch size, and 2x concurrent exports

Reply:
These are all set to the defaults, yes. See: [BatchConfig](https://docs.rs/opentelemetry_sdk/0.21.2/opentelemetry_sdk/trace/struct.BatchConfig.html)
>Set max_queue_size for [BatchConfig](https://docs.rs/opentelemetry_sdk/0.21.2/opentelemetry_sdk/trace/struct.BatchConfig.html). It’s the maximum queue size to buffer spans for delayed processing. If the queue gets full it will drops the spans. The default value of is 2048.

I'll add a comment making that explicit.

@guswynn Are you suggesting we set the default values here to 4x, or follow up with an override for 4x?